tests/unit/parser.js

```diff
@@ -6935,6 +6935,7 @@ exports.testStrictDirectiveASI = function (test) {
     .test("'use strict'\n(function fn() {})();", options);
 
   TestRun(test, 5)
+    .addError(2, "Missing \"use strict\" statement.")
```

Comment:
Because `;` isn't automatically inserted if the next token is `[`, so that code is like
```js
'use strict'[0] = '6';
```